Sources/SmithyHTTPAuth/CRTAdapters.swift

```diff
@@ -0,0 +1,103 @@
+//
```

Comment:
These are all adapter methods that convert types in `SmithyHTTPAuthAPI` to matching CRT types.